src/main/java/com/flansmod/common/teams/GameTypeTDM.java

```diff
@@ -39,49 +80,49 @@ public void roundCleanup() {
 
     @Override
     public void tick() {
-        if (autoBalance && time % autoBalanceInterval == autoBalanceInterval - 200 && needAutobalance()) {
-            TeamsManager.messageAll("\u00a7fAutobalancing teams...");
-        }
-        if (autoBalance && time % autoBalanceInterval == 0 && needAutobalance()) {
-            autobalance();
-        }
+//        if (autoBalance && time % autoBalanceInterval == autoBalanceInterval - 200 && needAutobalance()) {
```

Comment:
er, why comment all of this out?

Reply:
I forgot to remove this code earlier.
  this is necessary for the normal operation of the autobalancer